packages/pretty-format/src/__tests__/Immutable.test.js

```diff
@@ -11,7 +11,7 @@
 
 import React from 'react';
 import Immutable from 'immutable';
-import {getPrettyPrint} from './expect_util';
```

Comment:
The `getPrettyPrint` now only exports a single fn with the same name. Since there's no other exports there, maybe it's worth to change it to default export?

Reply:
Did the changes, not sure if that's what you meant, please take a look 👍